src/BsReactstrap__Alert.re

```diff
@@ -1,44 +1,57 @@
 [@bs.module "reactstrap"] external alert : ReasonReact.reactClass = "Alert";
 
-[@bs.obj]
-external makeProps : (
-  ~className: string=?,
-  ~closeClassName: string=?,
-  ~closeAriaLabel: string=?,
-  ~cssModule: 'a=?,
-  ~color: string=?,
-  ~isOpen: bool=?,
-  ~toggle: 'b=?,
-  ~tag: 'c=?,
-  ~transition: 'd=?,
-  unit
-) => _ = "";
+type toggle;
+type transition;
 
-let make = (
-  ~className=?,
-  ~closeClassName=?,
-  ~closeAriaLabel=?,
-  ~cssModule=?,
-  ~color=?,
-  ~isOpen=?,
-  ~toggle=?,
-  ~tag=?,
-  ~transition=?,
-  children
-) =>
+[@bs.deriving abstract]
+type props = {
+  [@bs.optional]
+  className: string,
+  [@bs.optional]
+  closeClassName: string,
+  [@bs.optional]
+  closeAriaLabel: string,
+  [@bs.optional]
+  cssModule: BsReactstrap__Props.cssModule,
```

Comment:
What do you think about using `open BsReactstrap__Props` to avoid long types?

Reply:
Sure can do. That'll clean it up.